src/store/CategoricalDisplayStore.js

```diff
@@ -92,6 +92,10 @@ export default class CategoricalDisplayStore {
 
     get context() { return this.chart.context; }
 
+    get height() {
```

Comment:
@shayanbinary it should be a computed property otherwise it's not observable and doesn't affect the component by changing.

Reply:
@morteza-binary as it's `height` and just change on mobile/screen, I didn't set `computed` for it.